ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -357,10 +357,9 @@ def set_action_for_agent(
         expected_shape = (spec.action_size,)
         if action.shape != expected_shape:
             raise UnityActionException(
-                f"The Agent {0} with BehaviorName {1} needs an input of dimension "
```

Comment:
This combination of an f-string with numbered .format() exposed a bug in pyupgrade: https://github.com/asottile/pyupgrade/issues/329